test/integration/lib.sh

```diff
@@ -102,6 +102,24 @@ kpatch_openEuler_dependencies()
 		"kernel-debuginfo-${kernel_version%.*}" "kernel-devel-${kernel_version%.*}"
 }
 
+kpatch_photon_dependencies()
+{
+	local flavor
+	flavor=$(uname -r | cut -d "-" -f 3)
+	if [[ $(uname -r | cut -d "-" -f 2) == "rt" ]]; then
+		flavor="rt"
+	fi
+
+	tdnf install -y kmod bash rpm-build coreutils gawk util-linux binutils sed findutils \
+		make gcc elfutils elfutils-devel systemd-rpm-macros build-essential
```

Comment:
nit:
`build-essentail` implicitly brings in `gcc, make, gawk, binutils`
`elfutils-devel` brings in `elfutils` so we can remove the redundant entries from here